### PR TITLE
Added a basic structure for the deal dashboard page

### PR DESCRIPTION
**Pull request by IonelLupu**

### CODE REVIEW

File: src/dealDashboard/dealClauses/dealClauses.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
+@import "src/resources/elements/primeDesignSystem/variables";
+
+$hover-transition-speed-fast: 175ms;
+$hover-transition-speed-medium: 0.3s;
+$hover-transition-speed-slow: 0.4s;
+
+p {
```
Comment (dkent600) [inline src/dealDashboard/dealClauses/dealClauses.scss:7]: Can you please put all of this css into a single container class?

File: src/dealDashboard/dealNavbar/dealNavbar.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { autoinject, bindable } from "aurelia-framework";
+import { DealTokenSwap } from "entities/DealTokenSwap";
+import "./dealNavbar.scss";
+
+@autoinject
+export class DealNavbar {
```
Comment (hiaux0) [inline src/dealDashboard/dealNavbar/dealNavbar.ts:6]: Wdyt about `DealMenubar`?

File: src/dealDashboard/dealClauses/dealClauses.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <header>
```
Comment (dkent600) [inline src/dealDashboard/dealClauses/dealClauses.html:2]: should the whole template be in a section?

File: src/dealDashboard/dealDescription/dealDescription.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.overview {
```
Comment (dkent600) [inline src/dealDashboard/dealDescription/dealDescription.scss:1]: Can you please put all of this css into a single container class?

File: src/dealDashboard/dealInfo/dealInfo.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<template>
+  <pcard>
```
Comment (dkent600) [inline src/dealDashboard/dealInfo/dealInfo.html:2]: should the whole template be in a section?

**Review by hiaux0 — COMMENTED** [review]
Looks good overal

File: src/dealDashboard/dealInfo/dealInfo.scss [hunk 6]
```diff
@@ -0,0 +1,4 @@
+pcard.pcard {
```
Comment (dkent600) [inline src/dealDashboard/dealInfo/dealInfo.scss:1]: Can you please put all of this css into a single container class?

File: src/dealDashboard/dealNavbar/dealNavbar.html [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<template>
```
Comment (dkent600) [inline src/dealDashboard/dealNavbar/dealNavbar.html:1]: should the whole template be in a section?

File: src/dealDashboard/dealVotes/dealVotes.html [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<template>
```
Comment (dkent600) [inline src/dealDashboard/dealVotes/dealVotes.html:1]: should the whole template be in a section?

File: src/dealDashboard/dealVotes/dealVotes.scss [hunk 9]
```diff
@@ -0,0 +1,4 @@
+pcard.pcard {
```
Comment (dkent600) [inline src/dealDashboard/dealVotes/dealVotes.scss:1]: Can you please put all of this css into a single container class?

**Review by dkent600 — CHANGES_REQUESTED** [review]
1. does the clause-related code still need to be in the dashboard viewmodel?
2. I note that the dashboard "isPrivate" computedFrom is wrong, and the computer property could be deleted altogether to just let others reference deal.registration.isPrivate.

File: src/dealDashboard/dealNavbar/dealNavbar.ts (hunk 2 above)
Comment (IonelLupu) [reply src/dealDashboard/dealNavbar/dealNavbar.ts:6]: Sounds good. yes

File: src/dealDashboard/dealClauses/dealClauses.scss (hunk 1 above)
Comment (IonelLupu) [reply src/dealDashboard/dealClauses/dealClauses.scss]: updated. I used the component itself as the container. No need for an additional element

File: src/dealDashboard/dealVotes/dealVotes.scss (hunk 9 above)
Comment (IonelLupu) [reply src/dealDashboard/dealVotes/dealVotes.scss]: updated. I used the component itself as the container. No need for an additional element

File: src/dealDashboard/dealVotes/dealVotes.html (hunk 8 above)
Comment (IonelLupu) [reply src/dealDashboard/dealVotes/dealVotes.html:1]: updated. I used the component itself as the container. No need for an additional element

File: src/dealDashboard/dealNavbar/dealNavbar.html (hunk 7 above)
Comment (IonelLupu) [reply src/dealDashboard/dealNavbar/dealNavbar.html]: updated. I used the component itself as the container. No need for an additional element

File: src/dealDashboard/dealInfo/dealInfo.scss (hunk 6 above)
Comment (IonelLupu) [reply src/dealDashboard/dealInfo/dealInfo.scss]: updated. I used the component itself as the container. No need for an additional element

File: src/dealDashboard/dealInfo/dealInfo.html (hunk 5 above)
Comment (IonelLupu) [reply src/dealDashboard/dealInfo/dealInfo.html:2]: updated. I used the component itself as the container. No need for an additional element

File: src/dealDashboard/dealDescription/dealDescription.scss (hunk 4 above)
Comment (IonelLupu) [reply src/dealDashboard/dealDescription/dealDescription.scss]: updated. I used the component itself as the container. No need for an additional element

**Review by dkent600 — APPROVED** [review]

**Pull request merged**